src/lib/dnssd/minimal_mdns/tests/TestResponseSender.cpp

```diff
@@ -312,36 +312,40 @@ void PtrSrvTxtMultipleRespondersToInstance(nlTestSuite * inSuite, void * inConte
 
 void PtrSrvTxtMultipleRespondersToServiceListing(nlTestSuite * inSuite, void * inContext)
 {
-    CommonTestElements common1(inSuite, "test1");
-    CommonTestElements common2(inSuite, "test2");
-
+    // CommonTestElements common1(inSuite, "test1");
+    // CommonTestElements common2(inSuite, "test2");
```

Comment:
Why was this changed to heap?

- should use std::unique_ptr instead of new/delete
- there should be a comment why we use heap here (since previously we did not seem to need it)